nupic/research/frameworks/dendrites/routing/utils.py

```diff
@@ -127,6 +133,10 @@ def train_dendrite_model(
     :param optimizer: optimizer object used to train the model
     :param device: device to use ('cpu' or 'cuda')
     :param criterion: loss function to minimize
+    :param context_model: a torch.nn.Module subclass which generates context vectors.
+                          If the context vector should be learned, then
+                          the contexts in the dataset are integers from which a context
+                          will be generated.
```

Comment:
In this description, it looks like you start line 3 before reaching the 88 character limit on line 2. If so, you should back up "the contexts ..." onto line 2.